xtrack/base_element.py

```diff
@@ -273,6 +273,8 @@ class BeamElement(xo.HybridClass, metaclass=MetaBeamElement):
     allow_backtrack = False
     skip_in_loss_location_refinement = False
     prebuilt_kernels_path = XT_PREBUILT_KERNELS_LOCATION
```

Comment:
I know that this is something that I wrote, but do we actually need this? I don't think so, but I wanted to check with you @freddieknets, I don't want to break something again. It seems that in a roundabout way it's used by the `PyMethod`s which then pass it to `compile_kernels` (could be used directly...), but in any case it's never anything but the default? Right?

Anyway, I added a little test, and all of them are passing.